scenarios/kubernetes_bazel.py

```diff
@@ -71,10 +71,15 @@ def main(args):
             res = 1
         else:
             try:
-                check(
-                    'bazel', 'run', '//:push-build', '--',
-                    '%s/%s' % (args.gcs, version)
-                    )
+                gcs_location = '%s/%s' % (args.gcs, version)
+                check('bazel', 'run', '//:push-build', '--', gcs_location)
```

Comment:
I'm not sure I understand. We've been computing and passing the gcs location this way, now we're just storing that in another location. Which part should be more generic?